src/libertem/api.py

```diff
@@ -635,7 +638,7 @@ def run_udf(self, dataset: DataSet, udf: UDF, roi=None, progress=False):
         """
         return UDFRunner(udf).run_for_dataset(dataset, self.executor, roi, progress=progress)
 
-    def map(self, dataset, f, roi=None):
+    def map(self, dataset, f, roi=None) -> BufferWrapper:
```

Comment:
Same comment as for `run_udf` applies here

Reply:
After running Context.map() in jupyter I am getting return type as  libertem.common.buffers.BufferWrapper.

Reply:
Ah, I was wrong here, `BufferWrapper` is correct!